llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1234,7 +1245,91 @@ static bool isUnsupportedAMDGPUAddrspace(Value *Addr) {
   return false;
 }
 
+static bool isUnsupportedSPIRAddrspace(Value *Addr, Function *Func) {
```

Comment:
The name of the function is confusing, because its body has almost no address-space related checks, but instead reacts to names of LLVM IR Values.

I think that some renaming should be done, or a comment added

Reply:
Makes sense, I changed it to `isUnsupportedSPIRAccess`